src/folio/users-api.ts

```diff
@@ -34,11 +34,28 @@ export default class UsersAPI extends FolioAPI {
     return blocks.manualblocks
   }
 
+  async getFeeFineActions(accountId) {
+    const urlParams = this.buildCqlQuery({ accountId: accountId })
+    const actions = await this.get(`/feefineactions`, { params: urlParams })
+    return actions.feefineactions
+  }
```

Comment:
I'd expect to see this in `feefines-api.ts` over `users-api.ts` — is the reason it's here because it takes an `accountId`?

Reply:
Thanks for reviewing. The reason is that `getAccounts` was already present here in the `users-api` and I am just adding to it. FeeFineActions are a property of a user's Account. I need to call this method from `getAccounts` below - I could import it from a different file I suppose.

Reply:
I'm not sure we are using this anywhere (re the `feefines-api`)

https://github.com/sul-dlss/folio-graphql/blob/81716488386d0067200fb45221f6ab447095f815/src/folio/index.ts#L397

Reply:
Also -- my understanding is that `/feefine` is general - i.e. a feefineId refers to a library-wide feefine Type. Not a specific user's fees. https://s3.amazonaws.com/foliodocs/api/mod-feesfines/r/feefines.html#feefines__feefineid__get